suites/nautilus/rgw/tier-1_rgw_multisite_test-upgrade-4.x-to-latest.yaml

```diff
@@ -0,0 +1,395 @@
+        ceph-rgw1:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_Mbuckets_with_Nobjects_download.yaml
+            timeout: 300
+      desc: test to create "M" no of buckets and "N" no of objects with download
+      module: sanity_rgw_multisite.py
+      name: Test download with M buckets with N objects
+      polarion-id: CEPH-14237
+
+  # Basic swift based tests
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_swift_basic_ops.py
+            config-file-name: test_swift_version_copy_op.yaml
+            timeout: 500
+      desc: test restoring of versioned objects in swift
+      module: sanity_rgw_multisite.py
+      name: test swift versioning copy
+      polarion-id: CEPH-10646
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_swift_basic_ops.py
+            config-file-name: test_swift_object_expire_op.yaml
+            timeout: 500
+      desc: test object expiration with swift
+      module: sanity_rgw_multisite.py
+      name: test swift object expiration
+      polarion-id: CEPH-9718
+
+  # DBR
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_multisite_dynamic_resharding_greenfield.yaml
+            verify-io-on-site: ["ceph-rgw1", "ceph-rgw2"]
+            timeout: 300
+            test-config:
+              user_count: 1
+              bucket_count: 1
+              objects_count: 100
+              dynamic_resharding: true
+              objects_size_range:
+                min: 15K
+                max: 500K
+              cluster_type: greenfield
+              max_objects_per_shard: 2
+              local_file_delete: true
+              test_ops:
+                create_bucket: true
+                create_object: true
+                download_object: false
+                delete_bucket_object: false
+                sharding:
+                  enable: false
+                  max_shards: 0
+                compression:
+                  enable: false
+                  type: zlib
+      desc: Reshading test - dynamic
+      module: sanity_rgw_multisite.py
+      name: test dynamic resharding on primary cluster
+      polarion-id: CEPH-83573596
+
+  - test:
+      clusters:
+        ceph-rgw1:
+          config:
+            script-name: test_Mbuckets_with_Nobjects.py
+            config-file-name: test_multisite_manual_resharding_greenfield.yaml
+            verify-io-on-site: ["ceph-rgw1", "ceph-rgw2"]
+            timeout: 300
+            test-config:
+              user_count: 1
+              bucket_count: 1
+              objects_count: 100
+              manual_resharding: true
+              shards: 997
+              objects_size_range:
+                min: 15K
+                max: 500K
+              cluster_type: greenfield
```

Comment:
Since this is 4.x upgrade suite, Instead of greenfield, we need to add brownfield(Not yet automated) scenario here